tutorials/ab3AddRemark.md

```diff
@@ -141,6 +168,53 @@ public class RemarkCommand extends Command {
 }
 ```
 
+<panel header="Update construction of `RemarkCommand` in `RemarkCommandTest`">
+
+- Since `RemarkCommand` now requires both an index and a remark for instantiation, the tests should be updated accordingly. Additionally, the execution of the command has been updated, so the tests should be adjusted to reflect these changes.
+- ```java
+    @Test
+    public void execute() {
+    final String remark = "Some remark";
```

Comment:
I am not too sure but should this be indented?

Reply:
Thanks! Added the indentation.